tests/integration/test_logtest/test_remove_old_sessions/test_remove_old_sessions.py

```diff
@@ -22,17 +24,23 @@
 configurations = load_wazuh_configurations(configurations_path, __name__)
 
 # Variables
+logtest_startup_timeout = 30
 wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
-logtest_sock = os.path.join(os.path.join(WAZUH_PATH, 'queue', 'sockets', 'logtest'))
-
+local_internal_options = {'analysisd.debug': str(1)}
 msg_create_session = """{"version":1, "command":"log_processing", "parameters":{
 "event": "Oct 15 21:07:56 linux-agent sshd[29205]: Invalid user blimey from 18.18.18.18 port 48928",
 "log_format": "syslog", "location": "master->/var/log/syslog"}}"""
 
```

Comment:
Use `json.dump` instead
```suggestion
create_session_data = {'version': 1, 'command': 'log_processing', 'parameters': {'event': 'Oct 15 21:07:56 linux-agent sshd[29205]: Invalid user blimey from 18.18.18.18 port 48928', 'log_format': 'syslog', 'location': 'master->/var/log/syslog'}}
msg_create_session = json.dumps(create_session_data)


```